ls1intum/Artemis
# `Development`: Remove critical inline styles

### Checklist
#### General



### Motivation and Context



For some users, Artemis might show an unstyled render for a few milliseconds because the bundled stylesheet is loaded  or used too late. Example by @krusche :
![grafik](https://user-images.githubusercontent.com/23171488/149638130-4bca444e-744f-4725-a38f-57831873d79d.png)

### Description

This was somehow caused by Angular's CSS optimization that inlines some of the most "critical" CSS, and fetches the rest of the CSS later. Apparently, this doesn't work correctly for our project, therefore, this PR disables this optimization.
The CSS bundle is now simply added as regular ``<link>`` tag.

In my tests, this doesn't seem to come with any performance drawbacks. Fetching of the CSS bundle starts at the same time as fetching the JS bundle now, and as the CSS is much smaller in kB than the JS bundle, it will be fully loaded even before Angular starts to render the first view.

@krusche verified that the weird rendering behavior at startup described above does not occurr anymore on this patch.

### Steps for Testing

Prerequisites:

1. Load the app using a force reload to bypass caches
2. Verify that everything renders as expected
3. In particular, make sure that no unstyled renders happen (if you ever experienced them, even for just a single frame).

### Review Progress





#### Code Review
#### Manual Tests
